### ISSUE TITLE
How did you calculate the weights used in the loss function?

### ISSUE THREAD

**Issue by YanMinbit** [PkuRainBow/OCNet.pytorch#47]
I find that you have used the weight = torch.FloatTensor([0.8373, 0.918, 0.866, 1.0345, 1.0166, 0.9969, 0.9754, 1.0489, 0.8786, 1.0023, 0.9539, 0.9843, 1.1116, 0.9037, 1.0865, 1.0955, 1.0865, 1.1529, 1.0507]) in the loss function to deal with the problem of class imbalance.  
Can you tell me how you got it? It doesn't look like the ratio of the number of pixels of classes. 
As I find the distribution of the number of pixels of classes mentioned in the article that introduces the cityscapes dataset is as follows:
![image](https://user-images.githubusercontent.com/17764899/56844538-4fabc480-68e4-11e9-889a-d2284d60a0de.png)
